Mixing/Base/src/SecondaryEventProvider.cc

```diff
@@ -52,7 +52,8 @@ namespace edm {
                                                  ProductRegistry& preg,
                                                  std::shared_ptr<ProcessConfiguration> processConfiguration)
       : exceptionToActionTable_(new ExceptionToActionTable),
-        workerManager_(std::make_shared<ActivityRegistry>(), *exceptionToActionTable_) {
+        // no type resolver for modules in SecondaryEventProvider for now
+        workerManager_(std::make_shared<ActivityRegistry>(), *exceptionToActionTable_, nullptr) {
```

Comment:
I left it out because it wasn't clear to me how to percolate the `ModuleTypeResolverMaker const*` to `SecondaryEventProvider` without resorting to global variables (or maybe it should be taken from the "process configuration" the module). I'll create an issue after we've settled with this PR.

Reply:
Opened the issue as https://github.com/cms-sw/cmssw/issues/40497